ring_of_fire_bot/model/ring_status.py

```diff
@@ -0,0 +1,9 @@
+from enum import Enum
+
+
+class STATUS(Enum):
+    WAITING_ON_PARTICIPANTS = 0
+    OPENING_CHANNELS = 1
+    BALANCED = 2
```

Comment:
should this be 'BALANCING' instead of 'BALANCED', what is the difference with the FINISHED status otherwise?
Not sure if we need some "initial" or "canceled" status, but we could always add those later if needed..